### PR TITLE
Fixed fallback of media file-version meta

### LINKED ISSUE #2066: Fallback for file version meta is not working correctly if description is null
The fallback for the description from file version meta data is not working correctly in the Sulu admin area if the description is null.

DB:
database

Sulu admin area:
admin

Frontend:
frontend

Note: This does not happen if the meta information is added in the Sulu admin area as an empty string is inserted as description if it is saved there. This is only important if files are imported with an import script.

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #2066
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

Fixes a small UI bug when you load fallbacks for a already loaded media model in javascript. For example if you load a media in de and the en-description is null then you will get the de-description when you load the en model afterwards.

#### Why?

Because of a reported issue (:

### CODE REVIEW

**Comment by danrot** [comment]
Your answer to the question "Why?" Is the exact reason why I am always a bit skeptical about predefined templates (for PRs in that case).
@webmozart What would you suggest to put into the  "Why?" paragraph, if there is already an issue describing the misbehavior in detail? Shouldn't that be optional?

@chirimoya @c00n84 IIRC we said that this behavior has to be changed, but should we already merge that now? As you've pointed out some people are already making use of that bug...

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-manager.js [hunk 1]
```diff
@@ -109,6 +111,11 @@ define([
                 promise.resolve(media.toJSON());
             } else {
                 media = Media.findOrCreate({id: mediaId});
+                if (clear) {
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-manager.js]: Do we really need a flag here? Is there any case where this behavior is what we want?

**Comment by wachterjohannes** [comment]
@danrot the flag was unnecessary and i have removed it.

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-manager.js [hunk 2]
```diff
@@ -109,6 +110,9 @@ define([
                 promise.resolve(media.toJSON());
             } else {
                 media = Media.findOrCreate({id: mediaId});
+                media.clear();
+                media.set({id: mediaId});
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-manager.js]: When reading these few lines... Why is the `findOrCreate` method called at all? We always fetch the data again anyway, so this seems wrong...

**Comment by webmozart** [comment]
@danrot see #2108

**Pull request merged**